### PR TITLE
More porting updates

### PR DESCRIPTION
**Pull request by bcoca**
##### SUMMARY

added new notes on callbacks
##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME

docs
##### ANSIBLE VERSION

```
2.4/2.5
```

### CODE REVIEW

File: docs/docsite/rst/porting_guide_2.4.rst [hunk 1]
```diff
@@ -97,6 +97,19 @@ Developers should start migrating from hardcoded inventory with dynamic inventor
 
 Both users and developers should look into the new plugins because they are intended to alleviate the need for many of the hacks and workarounds found in the dynamic inventory scripts.
 
+Callback plugins
+----------------
+
+Users:
+* Callbacks are now using the new configuration system, users should not need to change anything as the old system still works,
```
Comment (gundalow) [inline docs/docsite/rst/porting_guide_2.4.rst]: Newline here

https://github.com/bcoca/ansible/blob/more_porting_updates/docs/docsite/rst/porting_guide_2.4.rst#callback-plugins

File: docs/docsite/rst/porting_guide_2.4.rst [hunk 2]
```diff
@@ -97,6 +97,19 @@ Developers should start migrating from hardcoded inventory with dynamic inventor
...
+  but you might see a deprecation notice if any callbacks used are not inheriting from the built in classes. Developers need to update them as stated below.
+
+Developers:
+* If your callback does not inherit from ``CallbackBase`` (directly or indirectly via another callback), it will still work, but issue a deprecation notice.
```
Comment (gundalow) [inline docs/docsite/rst/porting_guide_2.4.rst]: Newline here

**Review by gundalow — CHANGES_REQUESTED** [review]

File: docs/docsite/rst/porting_guide_2.4.rst [hunk 3]
```diff
@@ -97,6 +97,19 @@ Developers should start migrating from hardcoded inventory with dynamic inventor
 
 Both users and developers should look into the new plugins because they are intended to alleviate the need for many of the hacks and workarounds found in the dynamic inventory scripts.
 
+Callback plugins
+----------------
+
+Users:
+* Callbacks are now using the new configuration system, users should not need to change anything as the old system still works,
+  but you might see a deprecation notice if any callbacks used are not inheriting from the built in classes. Developers need to update them as stated below.
```
Comment (gundalow) [inline docs/docsite/rst/porting_guide_2.4.rst]: s/but/however/

**Review by gundalow — APPROVED** [review]

**Comment by dharmabumstead** [comment]
This needs edits and should not have been merged. Please DO NOT merge documentation PRs until someone from the docs team has had a chance to review them. Thank you.

**Pull request merged**